### PR TITLE
[18.0][MIG] product_multi_company: Migration to 18.0

### PR DESCRIPTION
**Pull request by BertVGroenendael**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

**Review by chienandalu — CHANGES_REQUESTED** [review]
Rebase from branch 17.0 as it's just been merged with all the last features and fixes https://github.com/OCA/multi-company/pull/625

**Comment by BertVGroenendael** [comment]
In the last commits, I redid the migration starting from the 17.0 branch.

File: product_multi_company/hooks.py [hunk 1]
```diff
@@ -13,22 +11,27 @@
...
-            "domain_force": (
-                "['|', ('company_ids', 'in', company_ids),"
-                "('company_ids', '=', False)]"
-            ),
-        }
+def post_init_hook(env):
+    hooks.post_init_hook(
+        env,
+        "product.product_comp_rule",
+        "product.template",
```
Comment (chienandalu) [inline product_multi_company/hooks.py:18]: > In the last commits, I redid the migration starting from the 17.0 branch.

Are you sure, this changes should be already in v17...

File: product_multi_company/__manifest__.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "author": "Tecnativa," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/multi-company",
     "category": "Product Management",
-    "version": "16.0.1.0.1",
+    "version": "18.0.1.0.0",
```
Comment (chienandalu) [inline product_multi_company/__manifest__.py]: > In the last commits, I redid the migration starting from the 17.0 branch.

Are you sure? :thinking:

**Pull request closed without merging**